letsencrypt/templates/nginx/letsencrypt

```diff
@@ -0,0 +1,21 @@
+{% set domains = [] %}
```

Comment:
I don't have strong feelings, but I don't really see a need to suffix everything in `templates/` with .j2. It's mostly an edX convention, and I don't find it helpful tbh.